src/build/get_build_environment.mjs

```diff
@@ -23,9 +23,9 @@ import {getBuildNumber} from './get_build_number.mjs';
   Outputs:
   => the build environment object
 */
-export async function getBuildEnvironment(platform, buildMode) {
+export async function getBuildEnvironment(platform, buildMode, {sentryDSN} = {}) {
```

Comment:
I don't get the point of passing the sentryDSN here if one can already get it from the environment.

When will it not match the environment?

Can you remove sentryDsn from getBuildParameters?

Reply:
Good question. That needs more discussion. Perhaps we address that in a different PR?
In general, we want to minimize the flags and env vars. The only reason we have the sentry env var is because we don't want to hardcode and have forkers use it.